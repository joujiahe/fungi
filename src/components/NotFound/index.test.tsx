import { render } from "@testing-library/react";
import { NotFound } from "./index";

it("renders NotFound unchanged", () => {
  const { container } = render(<NotFound />);
  expect(container).toMatchSnapshot();
});
