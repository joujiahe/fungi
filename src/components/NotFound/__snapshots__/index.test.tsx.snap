// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders NotFound unchanged 1`] = `
<div>
  <div
    class="css-iddgbs"
  >
    <h2
      class="chakra-heading css-0"
    >
      Oops... not found
    </h2>
  </div>
</div>
`;
